tests/utils/ssh/ssh_win.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Exposes util to invoke remote commands on windows hosts via ssh.
+
+// +build runoncewin
```

Comment:
If I understand correctly, you are controlling to pick up during windows run. I would suggest to use different name avoid using "test" related tags here.